test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -87,6 +77,12 @@ public void parse_exitCommand_parsedCorrectly() {
         parseAndAssertCommandType(input, ExitCommand.class);
     }
 
+    @Test
+    public void parse_SortCommand_parsedCorrectly() {
+        final String input = "sort";
+        parseAndAssertCommandType(input, SortCommand.class);
```

Comment:
good, but perhaps you can add a test to verify that your sort command is working properly?